app/graphql/mutations/CreateConsignment.graphql

```diff
@@ -0,0 +1,11 @@
+mutation createCon ($name: String!, $seriesId: Int!){
```

Comment:
Maybe change mutation name to createConsignment

Reply:
changed